app/models/cart.rb

```diff
@@ -203,4 +203,9 @@ def total_price
       0.0
     end
   end
+
+  # may be replaced with paper-trail or similar at some point
```

Comment:
Since this may be the case, wondering if we should call the param `updated_at_i` or something so that we don't run into a problem distinguishing them down the road?